### ISSUE TITLE
[Bug Report] No default config directory while running from source on linux

### ISSUE THREAD

**Issue by juozaspo** [Amulet-Team/Amulet-Map-Editor#916]
### Current Behavior:

Trying to run on linux after compiling from source program tries to write to current directory. Depending on configuration and/or location from where it's been run can lead to all kinds of problems. Running from read only directory only causes it to crash.
### Expected behavior:

Program should have a default config directory, should not write to directory it's being run and/or detect if it is read-only.
### Steps To Reproduce:
- Run amulet from anywhere, notice config and logs directories are created in current directory instead of some default folder
- Run from read only directory such as root (`/`), have a crash instead, also crash log could not be saved because of permission error

### Environment:

- OS: Ubuntu Linux 22.04 LTS
- Minecraft Platform: Java
- Minecraft Version: not related to this issue
- Amulet Version: 0.10.11

### Additional context
```
(ENV) juozas@xubuntu:/$ python3 -m amulet_map_editor
INFO - PyMCTranslate Version 235
Traceback (most recent call last):
  File "/home/juozas/ENV/lib/python3.9/site-packages/amulet_map_editor/__main__.py", line 75, in main
    _init_log()
  File "/home/juozas/ENV/lib/python3.9/site-packages/amulet_map_editor/__main__.py", line 48, in _init_log
    os.makedirs(logs_path, exist_ok=True)
  File "/usr/lib/python3.9/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: './logs'

[Errno 13] Permission denied: './logs'
Traceback (most recent call last):
  File "/home/juozas/ENV/lib/python3.9/site-packages/amulet_map_editor/__main__.py", line 75, in main
    _init_log()
  File "/home/juozas/ENV/lib/python3.9/site-packages/amulet_map_editor/__main__.py", line 48, in _init_log
    os.makedirs(logs_path, exist_ok=True)
  File "/usr/lib/python3.9/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: './logs'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/juozas/ENV/lib/python3.9/site-packages/amulet_map_editor/__main__.py", line 95, in <module>
    main()
  File "/home/juozas/ENV/lib/python3.9/site-packages/amulet_map_editor/__main__.py", line 79, in main
    _on_error(e)
  File "/home/juozas/ENV/lib/python3.9/site-packages/amulet_map_editor/__main__.py", line 20, in _on_error
    with open("./logs/crash.log", "w") as f:
FileNotFoundError: [Errno 2] No such file or directory: './logs/crash.log'
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib/python3.9/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/config'
```

## Attachments

none
### Screenshots

none relevant
### Worlds

none

**Comment by gentlegiantJGC** [comment]
Yes I do need to fix this.

**Comment by gentlegiantJGC** [comment]
I think that should be everything changed that used a relative path.

**Comment by juozaspo** [comment]
Closing this as required features are implemented in in 0.10.16, works fine so far.
